src/Settings/components/transliteration.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { useState } from "react";
+import { Text } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon, Switch } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import {
+  setTransliteration,
+  toggleTransliteration,
+  TRANSLITERATION_LANGUAGES,
+} from "../../common/actions";
+import { useSelector } from "react-redux";
+import colors from "../../common/colors";
+import styles from "../styles";
+
+function TransliterationComponent({ isNightMode, dispatch }) {
+  const [isVisible, toggleVisible] = useState(false);
```

Comment:
instead of toggleVisible, you should use 'setVisible' as state setter function

Reply:
The toggle I have implemented serves a specific purpose. It's designed to set a boolean value to either true or false. Where as set is used to set any value.